src/panels/config/devices/device-detail/integration-elements/zwave_js/device-actions.ts

```diff
@@ -105,22 +105,28 @@ export const getZwaveDeviceActions = async (
       label: hass.localize(
         "ui.panel.config.zwave_js.device_info.update_firmware"
       ),
-      action: async () => {
-        if (
-          await showConfirmationDialog(el, {
-            text: hass.localize(
-              "ui.panel.config.zwave_js.update_firmware.warning"
-            ),
-            dismissText: hass.localize("ui.common.no"),
-            confirmText: hass.localize("ui.common.yes"),
-          })
-        ) {
-          showZWaveJUpdateFirmwareNodeDialog(el, {
-            device,
-            firmwareUpdateCapabilities,
-          });
-        }
-      },
+      action: isNodeFirmwareUpdateInProgress
```

Comment:
Since we don't subscribe to this, this could be outdated by the time someone clicks on the button, shouldn't we fetch it on click?